documents-d-archeologie-francaise.csl

```diff
@@ -105,7 +106,12 @@
   </macro>
   <macro name="responsability">
     <choose>
-      <if variable="author editor translator collection-editor" match="any">
+      <if variable="version">
+        <group suffix=" (éd.)">
```

Comment:
The difficulty here is that the style defined by the editor wants different labels that not necessarily correspond with the real role.
For instance the line quoted above has been added because computer programs authors need to be labelled as editors.